features/all-positions/AllSponsors.tsx

```diff
@@ -40,6 +42,27 @@ const PageButtons = styled.div`
   margin-bottom: 2em;
 `;
 
+const MoreInfo = styled.a`
```

Comment:
This dialog styling is something we should eventually pull out into its own module as this config is something we'll want again I'm sure. Outside scope of PR but worth mentioning